src/apps/loan-list/materials/utils/warning-bar.tsx

```diff
@@ -2,25 +2,27 @@ import React, { FC } from "react";
 import IconWarning from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-warning.svg";
 import { Link } from "../../../../components/atoms/link";
 import { getCurrentLocation } from "../../../../core/utils/helpers/url";
+import { useUrls } from "../../../../core/utils/url";
 
 interface WarningBarProps {
   linkText: string;
   overdueText: string;
 }
 
 const WarningBar: FC<WarningBarProps> = ({ linkText, overdueText }) => {
-  /* todo link til gebyrer */
-  const url = new URL("/", getCurrentLocation());
-
+  const { feesPageUrl } = useUrls();
+  const feesPage = new URL(feesPageUrl, getCurrentLocation());
```

Comment:
useUrl should deliver URL objects. Are you shure it is necessary to create your own object as well?

Reply:
ill fix